gips/scripts/process.py

```diff
@@ -28,6 +28,7 @@
 from gips import utils
 from gips.inventory import DataInventory
 from gips.inventory import orm
+from functools import reduce
```

Comment:
probably also needs to import `map` ; run with `--batchout` to test (perhaps add sys test)

Reply:
`map` is in the default namespace, but `reduce` is not...lame.